### PR TITLE
Test git, trailing newline in README

### PR DESCRIPTION
**Pull request by coding-ray-demo**
## What I did

1. Overwrite test.txt with my testing message, which is a general and verbose version of testing `git`.
1. Add a trailing newline in README.md

## Remarks
1. Printing README.md without a trailing newline inside makes output ugly in the last line. As  a result, please consider approve the change in README.md.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1 @@
-this tests git
+This is a test message
```
Comment (boray-tw) [inline test.txt]:
```suggestion
This is a testing message
```

Notice the grammar correctness.

Comment (coding-ray-demo) [reply test.txt]: Okay, corrected.

**Comment by coding-ray-demo** [comment]
Sorry, I accidentally closed it.

**Comment by boray-tw** [comment]
Good. Merged.

**Pull request merged**